### PR TITLE
#5796 [Manager] Save autofill from property will cause clean current data

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/appBar/createWorkspace.test.ts [hunk 1]
```diff
@@ -205,15 +205,15 @@ describe('Create Workspace', () => {
   });
 
   context('Volume', () => {
-    it('should able to add a volume', () => {
+    it.skip('should able to add a volume', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/createWorkspace.test.ts:208]: https://github.com/oharastream/ohara/pull/5792/files#r525834058

File: ohara-manager/client/src/components/Pipeline/PipelinePropertyDialog/PipelinePropertyForm.js [hunk 2]
```diff
@@ -105,4 +106,10 @@ PipelinePropertyForm.propTypes = {
...
 };
 
-export default PipelinePropertyForm;
+const areEqual = (prevProps, nextProps) => {
+  // The onSubmit handler passing down form the parent component will always be different
+  // from previous render, hance omitting from our comparison function
+  return isEqual(omitBy(prevProps, isFunction), omitBy(nextProps, isFunction));
+};
+
+export default memo(PipelinePropertyForm, areEqual);
```
Comment (konekoya) [inline ohara-manager/client/src/components/Pipeline/PipelinePropertyDialog/PipelinePropertyForm.js:115]: Fix the re-rendering issue after an "autofill" is copied.

File: ohara-manager/client/cypress/it/paper/schemaTable.test.ts [hunk 3]
```diff
@@ -0,0 +1,212 @@
+import * as generate from '../../../src/utils/generate';
+import { RecommendValue } from '../../../src/api/apiInterface/definitionInterface';
+import { SOURCE } from '../../../src/api/apiInterface/connectorInterface';
+import { KIND } from '../../../src/const';
+import { CellAction } from '../../types';
+
+type Data = {
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/paper/schemaTable.test.ts:7]: PropertyDialog spec has grown into like a monster spec...

File: ohara-manager/client/cypress/global.d.ts [hunk 4]
```diff
@@ -58,11 +58,6 @@ declare namespace Cypress {
       options?: CreateWorkspaceByApiOption,
     ) => Chainable<null>;
 
-    produceTopicData: (
```
Comment (konekoya) [inline ohara-manager/client/cypress/global.d.ts:61]: Unused

File: ohara-manager/client/cypress/support/customCommands.ts [hunk 5]
```diff
@@ -392,14 +392,19 @@ Cypress.Commands.add(
...
-  // if the intro dialog appears, we should close it
+Cypress.Commands.add('closeDialog', () => {
   cy.get('body').then(($body) => {
-    if ($body.find('[data-testid="intro-dialog"]').length > 0) {
-      cy.findByTestId('close-intro-button').filter(':visible').click();
+    const $dialogs = $body.find('[role="dialog"]');
+
+    if ($dialogs.filter(':visible').length > 0) {
+      // Using keyboard instead of a mouse click to avoid Cypress' detached DOM issue
+      cy.get('body').type('{esc}');
```
Comment (konekoya) [inline ohara-manager/client/cypress/support/customCommands.ts:401]: This is in fact not a good solution. I will revisit this later with a better approach

Comment (konekoya) [reply ohara-manager/client/cypress/support/customCommands.ts:401]: By "later" I mean within this PR :)

Comment (konekoya) [reply ohara-manager/client/cypress/support/customCommands.ts:401]: Hmm, since this command is now more generic, I will stick with this for now

**Review by eechih — APPROVED** [review]

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**